web/vite.config.ts

```diff
@@ -6,8 +6,7 @@ export default defineConfig({
   plugins: [react()],
   server: {
     proxy: {
-      "/api": {
-        // TODO: change this if you need a different port
+      "/web": {
```

Comment:
Why do you still use "/web" if you can append every endoint to "/"? Your application represents the main configuration. Therefore no "/web" is required.